header/sync/options.go

```diff
@@ -0,0 +1,72 @@
+package sync
+
+import (
+	"fmt"
+	"time"
+)
+
+type Options func(*Parameters)
+
+// Parameters is the set of parameters that must be configured for the syncer.
+type Parameters struct {
+	// BlockTime provides a reference point for the Syncer to determine
+	// whether its subjective head is outdated
+	BlockTime time.Duration
```

Comment:
BlockTime is a global network variable and something other than something users should configure independently. Let's keep it private and set it in the nodebuilder from p2p module where it's currently defined

Reply:
I see you already set it from modp2p. Let's just privatize this then